### ISSUE TITLE
Docker image tomcat-v1.2023.7 not working - no Dot executable

### ISSUE THREAD

**Issue by earthspike** [plantuml/plantuml-server#290]
**Describe the bug**
After updating the container image to `tomcat` ([tomcat-v1.2023.7](https://hub.docker.com/layers/plantuml/plantuml-server/tomcat-v1.2023.7/images/sha256-1310b3bafc81d99054b0ee18cade8d57046547e1274983acf6b9d41bace0821?context=explore)) the version output now states "Error: dot executable does not exist"

**To Reproduce**

1. Run the docker file. In my case this is: `docker run -d --restart=unless-stopped -p 8080:8080 --name "plantuml" -e BASE_URL=plantuml plantuml/plantuml-server:tomcat`
2. Browse `http://hostname:8080/plantuml/uml/AqijAixCpmC0`

![image](https://github.com/plantuml/plantuml-server/assets/1642011/90ca6def-0c00-436f-8111-412ba8c0ef18)

5. An error is also shown with `testdot`

**Expected behavior**
I expect to see the version panel without the error.

**Screenshots**
With `version`:

![image](https://github.com/plantuml/plantuml-server/assets/1642011/90ca6def-0c00-436f-8111-412ba8c0ef18)

With `testdot`:

![image](https://github.com/plantuml/plantuml-server/assets/1642011/2ad60ec2-80a4-49d8-982b-ba6c107098dd)

**Desktop (please complete the following information):**

- OS: Ubuntu 22.04 desktop. 
- Browser: Firefox
- Version: 113.0

**Additional context**
Container running on Ubuntu 22.04 server with Docker. [It's namespaced under `/plantuml` because I'm running it behind an Apache reverse proxy. The errors described above are from accessing the container directly, ie bypassing the proxy.]

**Comment by HansBug** [comment]
The same to me, we I enter the following code (exactly the same as the official example):

```
@startuml
skinparam rectangle<<behavior>> {
	roundCorner 25
}
sprite $bProcess jar:archimate/business-process
sprite $aService jar:archimate/application-service
sprite $aComponent jar:archimate/application-component

rectangle "Handle claim"  as HC <<$bProcess>><<behavior>> #Business
rectangle "Capture Information"  as CI <<$bProcess>><<behavior>> #Business
rectangle "Notify\nAdditional Stakeholders" as NAS <<$bProcess>><<behavior>> #Business
rectangle "Validate" as V <<$bProcess>><<behavior>> #Business
rectangle "Investigate" as I <<$bProcess>><<behavior>> #Business
rectangle "Pay" as P <<$bProcess>><<behavior>> #Business

HC *-down- CI
HC *-down- NAS
HC *-down- V
HC *-down- I
HC *-down- P

CI -right->> NAS
NAS -right->> V
V -right->> I
I -right->> P

rectangle "Scanning" as scanning <<$aService>><<behavior>> #Application
rectangle "Customer admnistration" as customerAdministration <<$aService>><<behavior>> #Application
rectangle "Claims admnistration" as claimsAdministration <<$aService>><<behavior>> #Application
rectangle Printing <<$aService>><<behavior>> #Application
rectangle Payment <<$aService>><<behavior>> #Application

scanning -up-> CI
customerAdministration  -up-> CI
claimsAdministration -up-> NAS
claimsAdministration -up-> V
claimsAdministration -up-> I
Payment -up-> P

Printing -up-> V
Printing -up-> P

rectangle "Document\nManagement\nSystem" as DMS <<$aComponent>> #Application
rectangle "General\nCRM\nSystem" as CRM <<$aComponent>>  #Application
rectangle "Home & Away\nPolicy\nAdministration" as HAPA <<$aComponent>> #Application
rectangle "Home & Away\nFinancial\nAdministration" as HFPA <<$aComponent>>  #Application

DMS .up.|> scanning
DMS .up.|> Printing
CRM .up.|> customerAdministration
HAPA .up.|> claimsAdministration
HFPA .up.|> Payment

legend left
Example from the "Archisurance case study" (OpenGroup).
See
====
<$bProcess> :business process
====
<$aService> : application service
====
<$aComponent> : application component
endlegend
@enduml

```

It acts like this

![1684195865798](https://github.com/plantuml/plantuml-server/assets/20508435/e62ccfbf-da0a-4c76-a91e-c195cc2e6bcb)

I'm using jetty-v1.2023.7. I'm sure this error can only be found in this version.

**Comment by kilaebi** [comment]
I found an obvious typo in the below line. I modified this line to `apt-get clean && \` and built it, and the application appears to be working fine!

https://github.com/plantuml/plantuml-server/blob/6c2bae2f8a3d58e8928878742ba7817af0471127/Dockerfile.jetty#L57

**Comment by delepster** [comment]
error introduced here: https://github.com/plantuml/plantuml-server/commit/5d7c4e1a032d4660809763d495975cd6cfdbc73d

plz fix this

**Comment by fgreinacher** [comment]
@arnaudroques I see this is already fixed on `master`. Would you please create a new release for this? 🙇

**Comment by earthspike** [comment]
This appears fixed in v1.2023.8 by 080cbaada904d300d4ff7d5fee9d0ee677ff8dea. Thank you!
